src/app/core/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,60 @@
+import { forwardRef, Ref, type ReactNode } from "react";
+import clsx from "clsx";
+import { LinkIcon } from "@/app/core/components/Icons/LinkIcon";
+
+export type TBadgeSize = "regular";
+
+interface IBadgeProps {
+  children: ReactNode;
+  icon: ReactNode;
+  outLink?: string;
+  size?: TBadgeSize;
+}
+
+export const BadgeSizes = {
+  regular: "px-1 pe-2 py-1 text-sm",
+};
+
+export const BadgeStyles =
+  "flex rounded-full items-center bg-transparent border border-gray-400 text-gray-700 dark:text-gray-300";
+
+const Badge = forwardRef(
```

Comment:
I'm not sure `forwardRef` is necessary here :thinking: 
It's used in the modal but it's needed there for the behaviour radix provides.